### PR TITLE
Update Files.de-DE.xlf

### PR DESCRIPTION
**Pull request by R3voA3**
- Updated translation to latest insider version
- Changed Alt+Ctrl+9 to Alt + Ctrl + 9

### CODE REVIEW

File: Files/MultilingualResources/Files.de-DE.xlf [hunk 1]
```diff
@@ -339,23 +339,23 @@
           <target state="translated">Dieser Ordner ist leer.</target>
         </trans-unit>
         <trans-unit id="NavBackButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Back (Alt+Left Arrow)</source>
+          <source>Back (Alt + Left Arrow)</source>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.de-DE.xlf:342]: Any reason for this change?

Comment (R3voA3) [reply Files/MultilingualResources/Files.de-DE.xlf]: This change in particular or all the shortcut related ones?

A space between the + looks nicer especially for shortcuts like Alt+Strg++ → Alt + Strg + +

Comment (yaichenbaum) [reply Files/MultilingualResources/Files.de-DE.xlf]: I think it's best to keep it without the space in order to stay consistent.

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@R3voA3 thank you!

**Pull request merged**